Gopkg.toml

```diff
@@ -1,3 +1,10 @@
+ignored = [
+  "gopkg.in/src-d/go-mysql-server.v0",
+  "gopkg.in/src-d/go-mysql-server.v0/sql",
+  "gopkg.in/src-d/go-mysql-server.v0/sql/expression",
+  "gopkg.in/src-d/go-mysql-server.v0/sql/plan",
+ ]
+
```

Comment:
Is this the thing to use our local `go-mysql-server` repo? Maybe we should open an issue to don't forget change it when we will have stable/tagged versions

Reply:
Added #13